docs/user/TerraformAzureAuthentication.md

```diff
@@ -21,13 +21,11 @@ To authenticate Terraform to access Azure, you will need to set the following fo
 | client_id | your app_id when using a Service Principal | string | "" |
 | client_secret | your client secret when using a Service Principal| string | ""|
 
-TF_VAR_name environment variables for these would be
+Once authenticated to the `az cli`, the following [bash (code) script](../../files/TerraformEnvVariableAssignment.sh) can be used to (re)assign the TF Environment Variables:
 
-```
-TF_VAR_tenant_id=00000000-0000-0000-0000-000000000000
-TF_VAR_subscription_id=00000000-0000-0000-0000-000000000000
-TF_VAR_client_id=00000000-0000-0000-0000-000000000000
-TF_VAR_client_secret=00000000-0000-0000-0000-000000000000
+```bash
+# source the bash script, presuming the file path
+source {path-to-file}TerraformEnvVariableAssignment.sh
```

Comment:
Remove {path-to-file}. its confusing. Also, can we rename the file? All lowercase and underscores. Something like tf_env_helper.sh